backend/src/common/constants/group.constants.ts

```diff
@@ -0,0 +1,3 @@
+const EMPTY_ARRAY_LENGTH = 0;
```

Comment:
```suggestion
```
you don't need constant for this. you can check this way `const hasItems = Boolean(items. lenght)`